WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginMagicLinkSentImprovedFragment.java

```diff
@@ -1,151 +0,0 @@
-package org.wordpress.android.login;
-
-import android.content.Context;
-import android.os.Bundle;
-import android.view.LayoutInflater;
-import android.view.Menu;
-import android.view.MenuInflater;
-import android.view.MenuItem;
-import android.view.View;
-import android.view.ViewGroup;
-import android.widget.Button;
-import android.widget.TextView;
-
-import androidx.annotation.Nullable;
-import androidx.appcompat.app.ActionBar;
-import androidx.appcompat.app.AppCompatActivity;
-import androidx.appcompat.widget.Toolbar;
-import androidx.fragment.app.Fragment;
-
-import javax.inject.Inject;
-
-import dagger.android.support.AndroidSupportInjection;
-
-public class LoginMagicLinkSentImprovedFragment extends Fragment {
```

Comment:
Migrating this to Kotlin using newer language and tools felt ---> Back to the future [LoginMagicLinkSentImprovedFragment](https://github.com/woocommerce/woocommerce-android/pull/7740/files#diff-9b576a79aed5c1efb573e8698c577749e8df56664b41ef440607948dc6504f71)

![](https://media.giphy.com/media/RmfhMeDyN0bVYBFKNR/giphy.gif)